web_01.html

```diff
@@ -0,0 +1,225 @@
+                                <h1>Clean Sketch</h1>
+                                <p>creative</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/antic_popcorn.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Antic Popcorn</h1>
+                                <p>illustration</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/website_fonts.jpg"></img>   
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Website Fonts</h1>
+                                <p>typography</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/vectorized_ideas.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Vectorized Ideas</h1>
+                                <p>illustration</p>
+                            </div>
+                        </div>
+                        <div class="project__content">
+                            <a href="#" class="project__image">
+                                <img src="/images/logo_mockup.jpg"></img>    
+                            </a>
+                            <div class="project__texts">
+                                <span class="left__border"></span>
+                                <span class="right__border"></span>
+                                <h1>Logo Mockup</h1>
+                                <p>creative</p>
+                            </div>
+                        </div>
+                    </div>
+                    <a href="#" class="project__button">SHOW ME MORE</a> 
+                </div>
+            </section>
+            <section class="parallax__background">
+                <p>Proudly serving over 1600 satisfied users!</p> 
+            </section>
+        </main>
+        <footer>
+            <div class="main__footer">  
```

Comment:
You can use `footer` as selector and `main__footer` can be omitted since it is the only child of footer.